submissions/vakulinina/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="main.css" rel="stylesheet">
+  <title>Hooli Popup</title>
+</head>
+
+<body>
+  <nav>
+    <ul class="navbar">
+      <li><a href="#">Mail</a></li>
+      <li><a href="#">Images</a></li>
+      <li>
+        <input class="app-button" type="checkbox" id="app-button">
+        <label class="app-button-label" for="app-button"><img src="icons/popup-button.png" alt="Apps"></label>
+        <section class="app-menu">
+          <a href="index.html"><img src="icons/calendar.png" alt="Calendar">Calendar</a>
```

Comment:
When we are using `section` we must add tag `h-(1-6)`.
And in this case `section` not very good fit. For semantics be better wrap all it's block in `ul` | `ol` > `li`

Reply:
What about `div`? Is it ok to use it instead of `section` here?

Reply:
Yes, of course, but inside `div` you must wrap your code like i wrote below: 
> For semantics be better wrap all it's block in `ul` | `ol` > `li`.

In this case we'r practice for [ally](https://medium.com/web-standards/pragmatic-a11y-rules-b16dd2f77685), so don't forget about semantic

Reply:
Done